.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: MicroProfile JWT Build
+
+on:
+  push:
+    paths-ignore:
+      - 'editorconfig'
+      - '.gitattributes'
+      - '.gitignore'
+      - 'CONTRIBUTING*'
+      - 'CODEOWNERS'
+      - 'KEYS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+      - 'site.yaml'
+  pull_request:
+    paths-ignore:
+      - 'editorconfig'
+      - '.gitattributes'
+      - '.gitignore'
+      - 'CONTRIBUTING*'
+      - 'CODEOWNERS'
+      - 'KEYS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+      - 'site.yaml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    name: build with jdk ${{matrix.java}}
+
+    steps:
+      - uses: actions/checkout@v2
+        name: checkout
+
+      - uses: actions/setup-java@v1.3.0
+        name: set up jdk ${{matrix.java}}
+        with:
+          java-version: ${{matrix.java}}
+
+      - name: build with maven
+        run: mvn verify
```

Comment:
@radcortez Hi Roberto, I believe that should include `clean` to get the osgi versioning plugin checking things

Reply:
You don't need it, because GH Actions build starts fresh every time, meaning empty Maven repo and no previous target folder.

Reply:
I recall Emily saying something about `clean' related to that plugin, but sure, we can add if needed

Reply:
Most likely, because Jenkins keeps the state of the previous build and when you build again, you have the old build version there, but GH Actions just start everything in a new container.  On each build, there are no old binaries and even all the Maven Dependencies are downloaded again, so I think we don't need clean.